### DISCUSSION TITLE
Type error: 'ThemeUIStyleObject' is declared but its value is never read.

### DISCUSSION THREAD

**Discussion by deadcoder0904** [system-ui/theme-ui#1217] [Q&A]
This used to work previously:

```ts
import { ThemeUIStyleObject } from '@theme-ui/css'

export const ScreenReader: ThemeUIStyleObject = {
	position: 'absolute',
	width: '1px',
	height: '1px',
	padding: '0',
	margin: '-1px',
	overflow: 'hidden',
	clip: 'rect(0,0,0,0)',
	border: '0',
}
```

But now it throws the above error: `Type error: 'ThemeUIStyleObject' is declared but its value is never read.`

Furthermore, when I hover over it inside VSCode, it says `Cannot use namespace 'ThemeUIStyleObject' as a type.ts(2709)`

After I remove it, I get another error which is similar: `Type error: 'Theme' is declared but its value is never read.` & it used to work as before you can see https://github.com/system-ui/theme-ui/issues/834#issuecomment-659350912

How do I make sure I don't get an error & get TS support?

**Comment by hasparus** [comment]
Hey @deadcoder0904, what version of Theme UI are you using?

**Reply by deadcoder0904** [reply]
Here are all my dependencies:

```json
"dependencies": {
		"@deadcoder0904/react-typical": "^0.1.2",
		"@emotion/core": "^10.0.35",
		"@mapbox/rehype-prism": "^0.5.0",
		"@mdx-js/react": "^1.6.18",
		"@theme-ui/components": "^0.4.0-rc.1",
		"@theme-ui/core": "^0.4.0-rc.1",
		"@theme-ui/css": "^0.3.1",
		"@theme-ui/mdx": "^0.4.0-rc.1",
		"@theme-ui/theme-provider": "^0.4.0-rc.1",
		"@types/nprogress": "^0.2.0",
		"fast-glob": "^3.2.4",
		"minireset.css": "0.0.6",
		"next": "9.5.5",
		"nprogress": "^0.2.0",
		"react": "^16.14.0",
		"react-dom": "^16.14.0",
		"tinytime": "^0.2.6"
	},
	"devDependencies": {
		"@mdx-js/loader": "^1.6.18",
		"@next/bundle-analyzer": "^9.5.5",
		"@types/mdx-js__react": "^1.5.3",
		"@types/node": "^14.11.10",
		"@types/react": "^16.9.53",
		"@types/react-dom": "^16.9.8",
		"@types/webpack-env": "^1.15.3",
		"cross-env": "^7.0.2",
		"feed": "^4.2.1",
		"file-loader": "^6.1.1",
		"rimraf": "^3.0.2",
		"simple-functional-loader": "^1.2.1",
		"typescript": "4.0.3"
	},
```

**Edit:** I see now that `@theme-ui/css` isn't using `@next` version, I'll update & report my findings :)

**Edit 2:** Nope same error. I even restarted TS server in VSCode. All `theme-ui` packages are on `^0.4.0-rc.5` now :)

**Comment by hasparus** [comment]
Okay, no clue here. I installed the same deps, and I don't have the error.
![image](https://user-images.githubusercontent.com/15332326/99509797-89409680-2986-11eb-81a6-214c31e718d9.png)

@atanasster, have you encountered a similar problem?

**Comment by atanasster** [comment]
No, never had this issue, i am importing from `theme-ui`But this shouldnt be the issue . Maybe clean Up node_modules and reinstall?


`import { ThemeUIStyleObject } from 'theme-ui';`
